modules/glib/2.82.2.bcr.2/MODULE.bazel

```diff
@@ -0,0 +1,31 @@
+module(
+    name = "glib",
+    version = "2.82.2.bcr.2",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "libffi", version = "3.4.7")
+bazel_dep(name = "pcre2", version = "10.46-DEV")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "rules_python", version = "0.35.0")
+
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
+    name = "gvdb",
```

Comment:
The glib release tarball includes a copy of `gvdb`:

```
[glib-2.82.2]$ ls subprojects/gvdb/gvdb
gvdb-builder.c
gvdb-builder.h
gvdb-format.h
gvdb-reader.c
gvdb-reader.h
```

Is it possible to use that copy instead of downloading a snapshot from gitlab?

Reply:
Oh, I didn't know about the subprojects. Sure, I changed it.